File: support_tools/swigwin-3.0.12/Examples/test-suite/javascript/dynamic_cast_runme.js
var dynamic_cast = require("dynamic_cast");

var f = new dynamic_cast.Foo();
var b = new dynamic_cast.Bar();

var x = f.blah();
var y = b.blah();

var a = dynamic_cast.do_test(y);
if (a != "Bar::test") {
  throw new Error("Failed.");
}

File: support_tools/swigwin-3.0.12/Examples/test-suite/ruby/dynamic_cast_runme.rb
#!/usr/bin/env ruby
#
# Put description here
#
# 
# 
# 
#

require 'swig_assert'

require 'dynamic_cast'

f = Dynamic_cast::Foo.new
b = Dynamic_cast::Bar.new

x = f.blah
y = b.blah

a = Dynamic_cast.do_test(y)
if a != "Bar::test"
  puts "Failed!!"
end
    
